ci-operator/step-registry/ipi/conf/azurestack/ipi-conf-azurestack-commands.sh

```diff
@@ -30,8 +30,6 @@ cp "/var/run/azurestack-cluster-secrets/service-principal" "${SHARED_DIR}/osServ
 cat >> "${CONFIG}" << EOF
 baseDomain: ppe.azurestack.devcluster.openshift.com
 credentialsMode: Manual
-metadata:
```

Comment:
is `metadata` what is causing the failure? this is valid key so I'm confused why it's failing

Reply:
It's being set twice so removing it from here.